README.md

```diff
@@ -1,5 +1,8 @@
-1. schema.sql/seed.sql is the blog database
+<h1>Blog DataBase</h1>
```

Comment:
A handy trick, in case you don't want to write full HTML, is to write `# Blog Database` (which gets interpreted as `<h1>Blog Database</h1>` :)